docs/index.rst

```diff
@@ -0,0 +1,18 @@
+.. include:: ../README.rst
+
+.. include:: ../TODO.rst
+
+Contents:
+=========
+
+.. toctree::
+   :maxdepth: 1
+
+   installation
+   usage
+   contributing
+   authors
+   changelog
+   todo
+   modules
```

Comment:
Yes, locally you generate it with `make docs` -- this command runs `sphinx-apidoc` which has a couple of options but basically it scans everything in the `kolibri` package. This generates the appropriate files for each package using the syntax you can find here:

http://www.sphinx-doc.org/en/stable/man/sphinx-apidoc.html

Sphinx can do the above-mentioned, but what the `sphinx-apidoc` script does is to generate a file for each package and module until a certain depth. This is where the inspiration and origins of the script apparently took place: http://stackoverflow.com/questions/2701998/sphinx-autodoc-is-not-automatic-enough